docs/contrib.rst

```diff
@@ -77,12 +108,17 @@ Contributing your changes
 
 .. todo::
 
-   Write this section
+   This section is unfinished
 
+At the moment there is no formal style guide for source code. We ask that you do your best effort to keep the code that you add or modify similar in style as well as in spirit to the code surrounding it, and we may ask you to change it during review. When in doubt, submit your code as-is.
 
-Proposing new features
-======================
 
-.. todo::
+Weekly meetings
+===============
+
+Every Monday at 17:00 UTC on our IRC channel `#amaranth-lang at libera.chat <irc>`_ or Matrix channel `#amaranth-lang@matrix.org <matrix>`_ (the channels are bridged together: the same messages appear on both), Amaranth maintainers meet with users and contributors to discuss newly submitted Requests for Comments and any other issues that warrant broad attention. These public meetings are the primary avenue of decision making.
```

Comment:
```suggestion
Every Monday at 17:00 UTC on our IRC channel `#amaranth-lang at libera.chat <irc>`_ or Matrix channel `#amaranth-lang@matrix.org <matrix>`_ (the channels are bridged together: the same messages appear on both), Amaranth maintainers meet with users and contributors to discuss newly-submitted Requests for Comment and any other issues that warrant broad attention. These public meetings are the primary avenue of decision making.
```

Reply:
The singular is "Request for Comments"--wouldn't the plural be "Requests for Comments"?

Reply:
Ah, I see. Then, yes, you're right. Often it's Request for Comment, so that's what tripped me up.